src/contexts/ServerContext.js

```diff
@@ -3,6 +3,9 @@ import PropTypes from 'prop-types';
 import axios from 'axios';
 import { useEnvironment } from './';
 import { irodsVersionComparator } from '../utils';
+import newZoneReport from './new_zonereport.json'
+import oldZoneReport from './old_zonereport.json'
```

Comment:
let's get better names than old and new..  perhaps let's use the schema version numbers...

that means... v3 (old, first used with 4.0.3 and 4.1.0) and v4 (new, first used in 4.3.1).

Reply:
Hmm, these imports were on the outdated code, so I've removed them (since we don't need example json in the actual ZMT.

I'm happy to change the naming of this line:
```
let new_version = zone_report.data.zones[0].hasOwnProperty('icat_server') ? false : true;
```

To be something like `is_v4` as the variable name (but since both report `schema_version` v4 it might be confusing. Not sure if using `v9` and `v10` would be a good idea since that deviates from the iRODS server version.

Reply:
What is v9 and v10?

Just thinking ahead about when we update the schema version again, and this code might have to distinguish between 3 or 4 or 5 different versions (gah, I hope not, but the design should account for that elegantly).

We can use iRODS version numbers...

`410` and `431` should be clear enough to distinguish, and then we can have a comment with the 'first used' wording to explain to the next reader (or ourselves) in a few months.

Reply:
ohhh, the catalog schema version number...   hmmm, that feels like not quite the right thing either.

Reply:
Sounds good, I'll rename the variable to `let version = <num>`, where `<num>` is either `410` or `431`, with a comment about first use.